helm/tenant/values.yaml

```diff
@@ -1,189 +1,338 @@
+      #
+      # These settings can control the minimum and maximum resources requested for each pod.
+      # If no worker nodes can meet the specified requests, the Operator may fail to deploy.
       resources: { }
-      ## Configure Pod's security context
-      ## We recommend to skip the recursive permission change by using
-      ## fsGroupChangePolicy as OnRootMismatch because it can be pretty
-      ## expensive for larger volumes with lots of small files.
+      ###
+      # The Kubernetes `SecurityContext <https://kubernetes.io/docs/tasks/configure-pod-container/security-context/>`__ to use for deploying Tenant resources.
+      #
+      # You may need to modify these values to meet your cluster's security and access settings.
+      #
+      # We recommend disabling recursive permission changes by setting``fsGroupChangePolicy`` to ``OnRootMismatch`` as those operations can be expensive for certain workloads (e.g. large volumes with many small files).
       securityContext:
         runAsUser: 1000
         runAsGroup: 1000
         fsGroup: 1000
         fsGroupChangePolicy: "OnRootMismatch"
         runAsNonRoot: true
-      ## Configure container security context
+      ###
+      # The Kubernetes `SecurityContext <https://kubernetes.io/docs/tasks/configure-pod-container/security-context/>`__ to use for deploying Tenant containers.
+      # You may need to modify these values to meet your cluster's security and access settings.
       containerSecurityContext:
         runAsUser: 1000
         runAsGroup: 1000
         runAsNonRoot: true
-      ## Configure topology constraints
+      ###
+      #
+      # An array of `Topology Spread Constraints <https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/>`__ to associate to Operator Console pods.
+      #
+      # These settings determine the distribution of pods across worker nodes.
       topologySpreadConstraints: [ ]
-      ## Configure Runtime Class
+      ###
+      #
+      # The name of a custom `Container Runtime <https://kubernetes.io/docs/concepts/containers/runtime-class/>`__ to use for the Operator Console pods.
       # runtimeClassName: ""
-  ## Mount path where PV will be mounted inside container(s).
+  ###
+  # The mount path where Persistent Volumes are mounted inside Tenant container(s).
   mountPath: /export
-  ## Sub path inside Mount path where MinIO stores data.
-  ## WARNING:
-  ## We recommend you to keep the same mountPath and the same subPath once the
-  ## Tenant has been deployed over your different PVs.
-  ## This is because if you change these values once Tenant is deployed, then
-  ## you will end up with multiple paths for different buckets. So please, be
-  ## very careful to keep same value for the life of the Tenant.
+  ###
+  # The Sub path inside Mount path where MinIO stores data.
+  # .. warning::
```

Comment:
mentioned that quotes not rendering properly on PR comment https://github.com/minio/docs/pull/1060#discussion_r1384008160, but here the warning section looks different.

Probably would be already fixed here.